src/kapil/Exams/ReturnIndices.java

```diff
@@ -0,0 +1,43 @@
+/*Exam - 1 : 18th April 2021
+
+Program 1 :
+Given an array of integers nums and an integer target, return indices of the two numbers such that they add up to the target.
+
+Input: nums = [2,11,15,7,4], target = 9
+
+Output: [0,3]
+
+Output: Because nums[0] + nums[3] == 9, we return [0, 3].
+-------------------------------------------------------------------
+*/
+package kapil.Exams;
+
+public class ReturnIndices {
+	
+	int[] getIndicies(int[] num, int Targetnum1){ {
```

Comment:
Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files
Dipali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1003/files